config.example.wikimedia.yaml

```diff
@@ -81,45 +82,62 @@ templates:
                   - name: default.group.local
                     domains: /./
 
-      /{module:page_revisions}:
+      /{module:page_revisions}: &wp/sys/page_revisions
         x-modules:
             - name: page_revisions
               version: 1.0.0
               type: file
 
-      /{module:key_rev_value}:
+      /{module:key_value}: &wp/sys/key_value
```

Comment:
To clarify, neither of these modules create any resources on start-up, so the net result is the same